Board Meeting Minutes/CCC Board Minutes 2019-11-21.md

```diff
@@ -0,0 +1,92 @@
+**Governing Board Conference Call -- 7:00am PST**
+
+**Thursday 21 November 2019**
+
+1.  **Call to Order / Roll Call**
+
+    a.  **Premier Members of the CCC**
+
+        i.  Stephen Walli (Microsoft)\* Acting Chair
+
+        ii. Philippe Robin (Arm)\*
+
+        iii. Mike Bursell (Red Hat) \*
+
+        iv. Peixin Hou (Huawei)\*
+
+        v.  Mike Muomoto (Intel)\*
+
+        vi. Nelly Porter (Google)\*
+
+    b.  **Guests / Observers**
+
+        i.  Stephano Cetola (LF PM, Acting Secretary)
+
+        ii. Scott Nicholas (LF)
+
+        iii. Richard Searle (Fortanix)*
+
+        iv. Simon Johnson (Intel)*
+
+        v.  Yan Michalevsky (Anjuna)*
+
+    c.  **Not in attendance**
+
+        i.  Xiaoning Li -- (Alibaba)\*
+
+        ii. Sergio Leunissen (Oracle)\*
+
+        iii. Zhipeng Huang -- (Huawei)*
+
+        iv. Jesse Schrater (Intel)*
+
+        v.  Yongzheng Wu (Huawei)
+
+**\*voting member**
+
+2.  **Agenda Items & Motions**
+
+    a.  The board approves the minutes of the Oct 31, 2019 minutes.
+
```

Comment:
In between points (a) and (b), the agenda and records show a discussion of a resolution to adopt the 3 initial projects that the TAC reviewed and recommended approving.  https://zoom.us/rec/play/v5V_d-_9qG43EtbAtgSDC_V5W9S1fPqsh3RN-vMNyhrjVyRXOgfwZ7UTN-BLNQVOLwrGz0Lp8xLEz5DY?startTime=1574347957000&_x_zm_rtaid=CP_owJjeSOqTseXC-QQR2A.1598478504805.079f73336de3c8feccb70029d78cff1b&_x_zm_rhtaid=715 shows the discussion lasted about 13 minutes (00:30:20 - 00:43:23) but I didn't get any audio when I tried to play the recording, and I notice the minutes do not even say this was discussed let alone voted on.   I would like to request that the minutes be amended with the outcome of the board discussion here.

Reply:
Indeed, I went and listened to the recording and that motion was tabled but should be noted here. I will provide that text shortly.